### ISSUE TITLE
Message signing is not supported for hardware wallets.

### ISSUE THREAD

**Issue by Lohann** [Zondax/ledger-kusama#214]
I'm getting this error message from polkadot extension when trying to use https://polkasafe.xyz/ for multisig:
<img width="465" alt="Screenshot 2024-07-12 at 17 39 34" src="https://github.com/user-attachments/assets/75e61643-0534-48b0-94ca-a1c0ba7a46ca">

I have a **Ledger Nano X**, which according to the docs supports multisig:
https://github.com/Zondax/ledger-kusama?tab=readme-ov-file#multisig

:link: [zboto Link](https://app.clickup.com/t/8695337za)

**Comment by ftheirs** [comment]
Hey @Lohann, are you using polkadotjs? I just checked with Subwallet extension and it seems to be working fine. Keep in mind that the error that you showed is related to Signing raw messages. This feature is enable on the app since ~2 years but the extensions/wallets needs to enable it.

PS: you can install the new generic app that supports all pallets 😉

**Comment by Lohann** [comment]
No, I was trying to use this tool: https://polkasafe.xyz/ it think it request a signature in the login page, but I'm unable to login.

**Comment by Lohann** [comment]
> his feature is enable on the app since ~2 years but the extensions/wallets needs to enable it.

hmm, is this the expect mode that I need to enable on ledger? let me try

**Comment by ftheirs** [comment]
On the login screen, which method are you using to connect the Ledger device?
![Screenshot 2024-07-12 at 6 06 26 PM](https://github.com/user-attachments/assets/f6ddbf85-48e6-4768-8b0c-add5ea70efac)

**Comment by Lohann** [comment]
No success, also I can't find any option in the polkadot extension for enable signing raw messages..

> On the login screen, which method are you using to connect the Ledger device?

I'm using polkadot extension

> you can install the new generic app that supports all pallets 

What is that generic app? On ledger or is another browser extension? I'm using Kusama on Ledger.

**Comment by ftheirs** [comment]
I replied some info here:
https://github.com/Zondax/ledger-kusama/issues/213#issuecomment-2204025514

There is a new app on Polkadot ecosystem that is currently working for Polkadot, Kusama and the AssetHub chains and after an upgrade on every parachain, they will be supported as well. Already available in LedgerLive

I just checked with the last version from Kusama and Subwallet extension doesn't allow you to connect. It asks for the Migration app. However, if you install any of the new apps, it will work. I haven't checked but it should be working fine for Talisman too
